alpha/lib/model/Partner.php

```diff
@@ -1806,7 +1806,11 @@ public function setHtmlPurifierBaseListUsage($v)
 	
 	public function getDefaultLiveStreamSegmentDuration()
 	{
-		return $this->getFromCustomData( "default_live_stream_segment_duration", null , null );
+		if (PermissionPeer::isValidForPartner(PermissionName::FEATURE_DYNAMIC_SEGMENT_DURATION, $this->getId()))
+		{
+			return $this->getFromCustomData("default_live_stream_segment_duration", null, null);
```

Comment:
What if value is not set already, better supply default to the get function

Reply:
done added default value to getFromCustomData()